From bad867214bcb9e3babfb468d525993bc5bade415 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Tue, 03 Jun 2025 06:16:02 +0300
Subject: [PATCH] [SW-230537] gaudi3: fix compilation warning

This patch fixes the implicit declaration warning
for function boot_info_get_no_merge.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I9f0c310e4fae1f3af00c66836bc0a6cf85905754
---

diff --git a/arch/gaudi3/arc/asic.c b/arch/gaudi3/arc/asic.c
index 01721e4..4fc4d05 100644
--- a/arch/gaudi3/arc/asic.c
+++ b/arch/gaudi3/arc/asic.c
@@ -40,6 +40,7 @@
 #include <telemetry_desc.h>
 #include <telemetry_ops.h>
 #include <hl_devs.h>
+#include <arcboot_export_dev.h>
 
 #include "pcie_dbi_regs.h"
 
